src/isomorphic/classic/element/ReactElement.js

```diff
@@ -208,6 +208,12 @@ ReactElement.createElement = function(type, config, children) {
         'React.createElement(...): Expected props argument to be a plain object. ' +
         'Properties defined in its prototype chain will be ignored.'
       );
+
+      warning(
```

Comment:
@aweary 
1. I'm not sure how or with what tools I could do regression testing
2. Well, I broke this PR ><